extism_cli/__init__.py

```diff
@@ -586,11 +586,23 @@ def main():
                             import tomllib as toml
                         manifest = toml.loads(s)
             else:
-                manifest = {
-                    "wasm": [{
-                        "path": args.wasm
-                    }],
-                }
+                from urllib.parse import urlparse
+                pieces = urlparse(args.wasm)
+                if pieces.scheme.lower() in ('http', 'https'):
+                    import urllib.request
+                    f = urllib.request.urlopen(args.wasm)
+                    manifest = {
+                        "wasm": [{
+                            "data": f.read()
+                        }],
+                    }
```

Comment:
I could not get the HttpRequest variant to work in the manifest. might want to follow up and look at why. I think this is okay for the time being.

Reply:
```suggestion
                    manifest = {
                        "wasm": [{
                            "url": args.wasm
                        }],
                    }
```
^ This worked for me, what error are you getting?

Reply:
hmm okay, i think the docs are wrong then: https://extism.org/docs/concepts/manifest/
it says there is a top level `req` key. But looking at the code it seems to be flattened into the level above it.

Reply:
I'll follow up with doc change if needed

Reply:
i see, yeah looks like the docs are wrong. you can reference the generated jsonschema: https://github.com/extism/extism/blob/59101d80a6d80b44a7762d860c523cbb19d07f03/manifest/schema.json#L123